api/zenao/v1/zenao.proto

```diff
@@ -3,6 +3,8 @@ syntax = "proto3";
 package zenao.v1;
 option go_package = "github.com/samouraiworld/zenao/backend/zenao/v1;zenaov1";
 
+// import "polls/v1/polls.proto";
```

Comment:
To remove ?

Reply:
i will remove it for now, our homemade gno protoc developed by norman does not handle import
i'm working as we speak on a PR to handle this

Reply:
[5c6b194](https://github.com/samouraiworld/zenao/pull/282/commits/5c6b1943a794b2e0c915c5ed0e6794fbfc07e85a)

Reply:
if it's too hard (I'm thinking about the gnolit stuff), consider moving the polls proto content into zenao proto